dcpy/models/product/dataset/metadata.py

```diff
@@ -201,7 +201,7 @@ class Metadata(BaseModel, extra="forbid"):
     # Hold onto this for serialization, to avoid Pydantics reformatting of the metadata.yml
     _templated_source_metadata: str
 
-    def __init__(self, templated_source_metadata: str, **data):
+    def __init__(self, templated_source_metadata: str = "", **data):
```

Comment:
If we do this, should `write_source_metadata_to_file` below model dump in case of empty `self.templated_source_metadata`?

Reply:
yeah, good call. Will make that change.

Reply:
Just in case you didn't know - you can exclude defaults and nones in a model_dump. For exclude_defaults, not sure what it'd do if you instantiated something with the default manually specified, I assume it would be excluded as well but I hope not